crates/holochain/src/conductor/cell.rs

```diff
@@ -239,10 +244,19 @@ impl Cell {
     }
 
     pub(super) async fn dispatch_scheduled_fns(self: Arc<Self>, now: Timestamp) {
```

Comment:
Is there a reason all this code swallows errors? It seems like that decision should be made above this level